src/components/common/ButtonWithIcon.vue

```diff
@@ -33,6 +38,29 @@ export default {
       type: String,
       default: '',
     },
+  },
+  computed: {
+    classes() {
+      return `${this.disabled ? 'disabledIcon' : this.type + 'Icon'} ${this.size}}`
```

Comment:
I actully had a error in my computed property for the base button. It has been fixed in the main branch but you still have the error (one `}` too much at the end of the line).

```suggestion
      return `${this.disabled ? 'disabledIcon' : this.type + 'Icon'} ${this.size} `
```